Simplenote/src/androidTest/java/com/automattic/simplenote/integration/TagsActivityTest.kt

```diff
@@ -0,0 +1,248 @@
+package com.automattic.simplenote.integration
+
+import android.widget.EditText
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.MediumTest
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.automattic.simplenote.BaseUITest
+import com.automattic.simplenote.R
+import com.automattic.simplenote.TagsActivity
+import com.automattic.simplenote.utils.isToast
+import com.automattic.simplenote.utils.withRecyclerView
+import org.hamcrest.CoreMatchers
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@MediumTest
+class TagsActivityTest : BaseUITest() {
+
+    @Test
+    fun listTagsShouldShowCompleteListTags() {
+        createTag("tag1")
```

Comment:
I agree. Since most of the test data is the same, I have abstracted the launch and creation of test data to one method. I added a fix in 93d2d289a25eb1c867351404993a3fe99bd8dd44